server/apps/tasks.py

```diff
@@ -94,10 +91,11 @@ def send_to(doc, update=None, desk_id=None, stage_id=None, user_id=None):
             task['status'] = destination_stage['task_status']
 
     if update:
-        update['task'] = task
+        update.setdefault('task', {})
+        update.get('task').update(task)
         update['expiry'] = get_expiry(desk_or_stage_doc=calculate_expiry_from)
     else:
-        doc['task'] = task
+        doc.get('task', {}).update(task)
```

Comment:
Yes, it won't assign. It will update the task in the doc. The assumption is that the doc already has task or assigned in the first line of send_to `original_task = doc.setdefault('task', {})`